epub33/fxl-a11y/index.html

```diff
@@ -74,7 +74,9 @@
 
 Fixed Layout publications present some unique challenges for accessibility. The requirements laid out in [[EPUB-A11Y-11]] recommend [[WCAG2]] AA, but for many use cases in fixed layout, that might not be possible without fundamental changes to the content.
 
-We want to recognize these challenges for content creators, and in this document will outline some techniques for managing these challenges. We encourage content creators to explore the possibilities that digital publications present, even when creating fixed layout publications.
+In particular, the needs of people with low vision or learning disabilities that rely on the transformation of text are still almost impossible to accommodate in fixed layout content. Content creators concerned about these particular areas are recommended to review whether the content they are producing needs to be in fixed layout. 
```

Comment:
"are recommended to review" is a bit awkward. I wonder if a slightly stronger statement is warranted:

> Content creators concerned about this may choose not to use fixed layout.